src/tests/test_filesystem_mounting.py

```diff
@@ -0,0 +1,181 @@
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide
+def test_filesystem_mounting(selenium):
+    import asyncio
+    from string import Template
+
+    from pyodide.code import run_js
+
+    PYODIDE1_FLAG = "pyodide1"
+    PYODIDE2_FLAG = "pyodide2"
+
+    # setup the pyodide objects in js and return them to the python code
+    setup_creation_in_js = Template(
+        """
+        async function set_pyodide_objects() {
+            let pyodide1 = await loadPyodide();
+            let pyodide2 = await loadPyodide();
+
+            await pyodide1.FS.mkdir("${PYODIDE1_DIR}");
+            await pyodide2.FS.mkdir("${PYODIDE2_DIR}");
+
+            await pyodide2.FS.mount(
+                pyodide2.FS.filesystems.PROXYFS,
+                {
+                    root: "${PYODIDE1_DIR}",
+                    fs: pyodide1.FS,
+                },
+                "${PYODIDE2_DIR}"
+            );
+            self.pyodide1 = pyodide1;
+            self.pyodide2 = pyodide2;
+            return [pyodide1, pyodide2];
+        }
+        set_pyodide_objects();
+        """
+    )
+
+    # setup the python code to write to the file in shared directory mounted in both pyodide objects
+    # this code is executed in both pyodide objects
+    setup_creation_in_python = Template(
+        """
+        import os
+
+        with open("${MOUNT_DIR}/file_from_${FILE_SUFFIX}.txt", "w", encoding="utf-8") as f:
+            f.write("${CONTENT}")
+        """
+    )
+
+    # test the python code to read the file in shared directory mounted in both pyodide objects
+    # this code is executed in both pyodide objects
+    test_creation_in_python = Template(
+        """
+        # Test os.exists
+        assert os.path.exists("${MOUNT_DIR}/file_from_${FILE_SUFFIX_1}.txt") is True
+        assert os.path.exists("${MOUNT_DIR}/file_from_${FILE_SUFFIX_2}.txt") is True
+        assert os.path.exists("${MOUNT_DIR}/file_from_nowhere.txt") is False
```

Comment:
nit:

```suggestion
assert os.path.exists("${MOUNT_DIR}/file_from_${FILE_SUFFIX_1}.txt")
assert os.path.exists("${MOUNT_DIR}/file_from_${FILE_SUFFIX_2}.txt")
assert not os.path.exists("${MOUNT_DIR}/file_from_nowhere.txt")
```

assertion check for truthy/falsy values can be shortened.

Reply:
Sure, that's no problem

Reply:
Oh, I mean I prefer what you recommended 😀